### DISCUSSION TITLE
Error in installation of FFTW-3.3.10 on M2 MacBook Pro

### DISCUSSION THREAD

**Discussion by channes1** [Homebrew/discussions#4574] [Getting started]

**Comment by carlocab** [comment]
Since you're compiling FFTW yourself, this doesn't look related to Homebrew. Please find a more appropriate venue to ask for help.

**Reply by channes1** [reply]
I have tried with both methods. However, I was not able to install it. Check the error for details.

**Reply by carlocab** [reply]
Ok. Please provide the information requested in the template so that people who read this might actually be able to help you.
